src/ukrdc_cupid/core/audit/validate_matches.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
+from sqlalchemy.orm import Session
+import ukrdc_sqla.ukrdc as orm
+from sqlalchemy import select, and_
+from typing import List
+
+def validate_demog(session: Session, dob:datetime, pid:str):
+    """Function to validate a date of birth against a patient record
+
+    Args:
+        session (Session): _description_
+        dob (datetime): _description_
+        pid (str): _description_
+    """
+
+    dob_query = select(orm.Patient.birthtime).where(orm.Patient.pid == pid).group_by(orm.Patient.birthtime)
```

Comment:
Unless i'm missing something there should only be 1 Birthtime per PID - so why is it doing a group_by?